controller/requirements.txt

```diff
@@ -1,4 +1,4 @@
-grpcio==1.20.1
+grpcio==1.41.1
 grpcio-tools==1.20.1
```

Comment:
could we align the grpcio-tools version while we're at it?

Reply:
what do you mean?

Reply:
```suggestion
grpcio==1.41.1
grpcio-tools==1.41.1
```

Reply:
done